web/.env

```diff
@@ -1 +0,0 @@
-REACT_APP_API_URL=http://localhost:8080/
```

Comment:
i left this file empty because i thought maybe environment variables will still be needed in the future?

maybe this file should only exist for local dev though, so maybe we shouldn't commit it. for prd there are other more secure ways of uploading environment variables

i'm going to delete this file but keep all the dependencies on dotenv